.gitignore

```diff
@@ -3,3 +3,5 @@ dist/
 node_modules
 .DS_Store
 .vscode
+coronadatascraper-cache/*
+coronadatascraper-cache
```

Comment:
This should be left out, `coronadatascraper-cache` is the submodule we use to store the cache for this project.

Reply:
My bad, I had not meant to include it initially.